app/topics/topic-management/add-existing-subtopic/add-existing-subtopic.component.ts

```diff
@@ -36,19 +38,31 @@ export class AddExistingSubtopicComponent {
     this.showChange.emit(this.show);
   }
 
-  public searchTopics() {
+  private filterTopics(topics:any) {
```

Comment:
please let space between variable name and variable type. Also you are getting topics, so it can be `filterTopics(topics: Topic)`